hassio-google-drive-backup/backup/static/layouts/partials/error-messages.jinja2

```diff
@@ -653,8 +653,8 @@
 {% call macros.errorMessage("Google drive Returned an Unexpected Response", "google_unexpected", "error", google_unexpected_actions) %}
   <p>
     The addon was unable to communicate with Google Drive because it gave back an unreadable response.  
-    Withotu beign able to communicate with google drive, the addon will nto be able to back up backups to the cloud.
-    This usually resolves itself after the addon retrys, but if you see this error stick around for a while you can try:
+    Without being able to communicate with Google Drive, the addon will not be able to transfer backups to the cloud.
```

Comment:
Good call wrt "transfer".  I've struggled to change all the relevan6t language in the add-on after they renamed "snapshots" to "backups"